docs/docs/cmd/purview/retentionevent/retentionevent-get.md

```diff
@@ -0,0 +1,129 @@
+# purview retentionevent get
+
+Get a retention event
+
+## Usage
+
+```sh
+m365 purview retentionevent get [options]
+```
+
+## Options
+
+`-i, --id <id>`
+: The Id of the retention event.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Get a retention event by id
+
+```sh
+m365 purview retentionevent get --id c37d695e-d581-4ae9-82a0-9364eba4291e
+```
+
+## Remarks
+
+!!! attention
+    This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+## More information
+
+This command is part of a series of commands that have to do with event-based retention. Event-based retention is about starting a retention period when a specific event occurs, instead of the moment a document was labeled or created.
+
+[Read more on event-based retention here](https://learn.microsoft.com/en-us/microsoft-365/compliance/event-driven-retention?view=o365-worldwide)
+
+
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+        "displayName": "Employee Termination",
```

Comment:
Let's use the 'Contract Expiry Event' example that we use in the other commands as well. The displayName for such an event might read something like 'Contract xyz expired' and description: 'The retention period dor documents related to Contract xyz started because the contract expired.'